Ducky-Chickens/bangazon-API-sprint1
# Created Employee Computer Join Table, Updated README with ERDs, Added POST method to Computer routing

**Changes**:
Computer route update, GET single and POST route methods, ERD folder with links to images in README, employee_computer files in generators and databases folder, update to build-db.js

**Testing**:
run ```npm run build-tables``` // To check out new employee_computer table
run ```nodemon app.js``` // To view single computer in browser with api/computers/:id

Postman POST instructions:

  - Headers
    - 'Key' : Content-Type, 
    - 'Value' : application/json

  - Body 
    - Select 'Raw'
    - Insert following object -


```
      {
        "purchaseDate": "YYYY/MM/DD"
      }
```

**Expected results**:
- User should be able to: run ```npm run build-tables``` and see employee_computer table
- User should be able to POST and GET from Postman for Computers table

Relates to #8
